From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Mon, 27 Jul 2026 23:17:07 +0200
Subject: brcmfmac: Demote informational boot messages to debug level

The firmware file selection, the optional txcap blob being absent, and
the firmware version dump are of no interest on a normal boot:

  brcmfmac: brcmf_fw_alloc_request: using brcm/brcmfmac43455-sdio ...
  brcmfmac: brcmf_c_process_txcap_blob: no txcap_blob available (err=-2)
  brcmfmac: brcmf_c_preinit_dcmds: Firmware: BCM4345/6 wl0: ...

Print them with brcmf_dbg(INFO, ...) instead, so they are still
available with the INFO debug level enabled.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c   | 4 ++--
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/firmware.c | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
index 111111111111..222222222222 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
@@ -208,7 +208,7 @@ static int brcmf_c_process_txcap_blob(struct brcmf_if *ifp)
 
 	err = brcmf_bus_get_blob(bus, &fw, BRCMF_BLOB_TXCAP);
 	if (err || !fw) {
-		brcmf_info("no txcap_blob available (err=%d)\n", err);
+		brcmf_dbg(INFO, "no txcap_blob available (err=%d)\n", err);
 		return 0;
 	}
 
@@ -372,7 +372,7 @@ int brcmf_c_preinit_dcmds(struct brcmf_if *ifp)
 	strsep(&ptr, "\n");
 
 	/* Print fw version info */
-	brcmf_info("Firmware: %s %s\n", ri->chipname, buf);
+	brcmf_dbg(INFO, "Firmware: %s %s\n", ri->chipname, buf);
 
 	/* locate firmware version number for ethtool */
 	ptr = strrchr(buf, ' ');
diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/firmware.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/firmware.c
index 111111111111..222222222222 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/firmware.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/firmware.c
@@ -834,7 +834,7 @@ brcmf_fw_alloc_request(u32 chip, u32 chiprev,
 	if (!fwreq)
 		return NULL;
 
-	brcmf_info("using %s for chip %s\n",
+	brcmf_dbg(INFO, "using %s for chip %s\n",
 		   mapping_table[i].fw_base, chipname);
 
 	mp_path = brcmf_mp_global.firmware_path;
-- 
Armbian

